test/common.yml

```diff
@@ -23,6 +23,9 @@ serviceSubnet: 100.100.0.0/16
 opa:
   version: 0.13.5
   kubeMgmtVersion: 0.8
+  bundle_url: https://discovery-opa-bundles.s3-eu-west-1.amazonaws.com
```

Comment:
we can't test in this repo with discovery assets, we can host a prepacked bundle in Github and then reference it via: https://raw.githubusercontent.com/